GRPC/grpcServer/grpcserver.go

```diff
@@ -2,20 +2,25 @@ package grpcServer
 
 import (
 	"context"
-	auth_proto "github.com/Baraulia/AUTHORIZATION_SERVICE/GRPC"
-	"github.com/Baraulia/AUTHORIZATION_SERVICE/pkg/logging"
+	"fmt"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/reflection"
 	"net"
+	auth_proto "stlab.itechart-group.com/go/food_delivery/authorization_service/GRPC"
```

Comment:
given the naming convention that go uses for its packages and variables, I would either call it `authproto` (because packages usually don't contain special characters) or `authProto` (similar to naming convention for regular variables).